code/modules/shuttles/marine_ferry.dm

```diff
@@ -74,10 +74,10 @@
 		return //Kill it so as not to repeat
 
 /datum/shuttle/ferry/marine/proc/load_datums()
-	if(!(info_tag in s_info))
+	if(!(info_tag in GLOB.s_info))
```

Comment:
doesnt seem like anything puts anything in this, could probbaly be cut and /proc/loadShuttleInfoDatums too

Reply:
duplicate